keps/sig-scheduling/5055-dra-device-taints-and-tolerations/README.md

```diff
@@ -268,11 +277,11 @@ type BasicDevice struct {
 }
 
 // DeviceTaintsMaxLength is the maximum number of taints per device.
-const DeviceTaintsMaxLength = 8
+const DeviceTaintsMaxLength = 4
```

Comment:
(just curious) why there's a max length and reduced? does node taints also have this kind of limitation?

Reply:
This is a result of rebalancing how large each Device in a ResourceSlice is allowed to be. It was a bit too large.

The limits for node taints are different, but I also would argue that they correspond more to DeviceTaintRules: many entities can add node taints or DeviceTaintRules. Device taints in a ResourceSlice can only be set by the DRA driver.